ansible/module_utils/multi_servers_utils.py

```diff
@@ -0,0 +1,63 @@
+class MultiServersUtils:
```

Comment:
looks to be pending on another PR to show the real diff, so let's make it clear in the PR description and we will wait until the previous PR is merged to review this one.

Reply:
What I usually do is mark it in title:

.... (Pending #xxx)

Reply:
thanks, awesome